src/runtime/classmanager.cs

```diff
@@ -39,6 +39,78 @@ public static void Reset()
             cache = new Dictionary<Type, ClassBase>(128);
         }
 
+        public static IList<ClassBase> GetManagedTypes()
+        {
+            return cache.Values.ToArray(); // Make a copy.
+        }
+
+        internal static void RemoveClasses()
+        {
+            var visited = new HashSet<IntPtr>();
+            var visitedHandle = GCHandle.Alloc(visited);
+            var visitedPtr = (IntPtr)visitedHandle;
+            try
+            {
+                foreach (var cls in cache.Values)
+                {
+                    cls.CallTypeTraverse(OnVisit, visitedPtr);
+                    // XXX: Force release instance resources but not dealloc itself.
+                    cls.CallTypeClear();
```

Comment:
Should this implement `IDisposable` ? Else I'd call it another name (like `DisposeType`) because it has meaning in C#

Reply:
`IDisposable` is not a fit option because it has no parameters, but it needs a callback and an argument for `tp_traverse` here.
About `DisposeType`, do you mean to implement it as a method for `ClassBase`? But it still needs arguments from outside...... So I'm not sure what's the best for here too.
btw. I updated the comments.